src/pages/Home/ui/Calendar/style.css.ts

```diff
@@ -0,0 +1,229 @@
+import { font } from 'shared/styles/font.css';
+import theme from 'shared/styles/theme.css';
+
+export const calendarContainer = style({
+  width: 'calc(100vw - 420px)',
+  height: '100vh',
+});
+
+export const calendarHeaderContainer = style({
+  display: 'inline-flex',
+  padding: '18px 16px 17px 0px',
+  alignItems: 'center',
+  justifyContent: 'space-between',
+  width: '100%',
+  height: '80px',
+});
+
+export const calendarHeaderMain = style({
+  display: 'flex',
+  alignItems: 'center',
+  height: '80x',
+  gap: '8px',
+});
+
+export const calendarHeaderPlusBtn = style({
+  position: 'relative',
+  display: 'flex',
+  width: '60px',
+  height: '36px',
+  padding: '5px 17px',
+  justifyContent: 'center',
+  alignItems: 'center',
+  borderRadius: '8px',
+  background: theme.white,
+});
+
+export const calendarHeaderSub = style({
+  display: 'flex',
+  alignItems: 'center',
+  gap: '16px',
+});
+
+export const calendarHeaderBtnLayout = style({
+  display: 'flex',
+  alignItems: 'center',
+  gap: '12px',
+});
+
+export const calendarHeaderBtn = style({
+  width: '26px',
+  height: '26px',
+  color: theme.black,
+});
+export const calendarHeaderTodayBtn = style({
+  color: theme.black,
+  ...font.p1,
+  cursor: 'pointer',
+});
+
+export const calendarTitleYear = style({
+  color: theme.black,
+  fontFamily: 'Pretendard',
+  ...font.H2,
+  fontWeight: '400',
+});
+export const calendarTitleMonth = style({
+  color: theme.black,
+  fontFamily: 'Pretendard',
+  ...font.H2,
+  fontWeight: '600',
+});
+
+export const calendarSearchBar = style({
+  display: 'flex',
+  width: '378px',
+  height: '40px',
+  padding: '7px 0px 7px 12px',
+  alignItems: 'center',
+  gap: '12px',
+  borderRadius: '8px',
+  border: `1px solid ${theme.gray[100]}`,
+  backgroundColor: theme.white,
+});
+
+export const calendarSearchText = style({
+  color: theme.black,
+  ...font.p2,
+  width: '100%',
+  '::placeholder': {
+    color: theme.gray[400],
+  },
+});
+
+export const calendarScheduleContainer = style({
+  display: 'flex',
+  height: '100%',
+  alignItems: 'center',
+  gap: '8px',
+  alignSelf: 'stretch',
+  flexShrink: '0',
+  borderRadius: '5px',
+  backgroundColor: theme.blue[50],
+  borderLeft: `5px solid ${theme.blue[500]}`,
+  padding: '0 8px',
+});
+
+export const calendarScheduleText = style({
+  ...font.p2,
+});
+
+export const calendarTodoContainer = style({
+  display: 'flex',
+  height: '100%',
+  alignItems: 'center',
+});
+
+export const calendarTodoText = style({
+  flex: '1 0 0',
+  ...font.p2,
+  overflow: 'hidden',
+  selectors: {
+    '&::before': {
+      content: '"•"',
+      fontSize: '1.2em',
+      marginRight: '6px',
+      color: theme.black,
+    },
+  },
+});
+
+globalStyle('.fc-theme-standard .fc-scrollgrid', {
+  borderRadius: '20px 20px 0px 0px',
+  borderTop: `1px solid ${theme.gray[100]}`,
+  borderRight: `2px solid ${theme.gray[100]}`,
+  borderLeft: `2px solid ${theme.gray[100]}`,
+});
```

Comment:
calendarContainer 아래 있는 className들만 제어되게 해주세요.

```suggestion
globalStyle('${calendarContainer} .fc-theme-standard .fc-scrollgrid', {
  borderRadius: '20px 20px 0px 0px',
  borderTop: `1px solid ${theme.gray[100]}`,
  borderRight: `2px solid ${theme.gray[100]}`,
  borderLeft: `2px solid ${theme.gray[100]}`,
});
```